locales/tr-TR/translation.json

```diff
@@ -0,0 +1,280 @@
+  "Expected fee": "Beklenen ücret",
+  "Address format is incorrect": "Adres biçimi yanlış",
+  "Select the address to send funds from.": "Paranın gönderileceği adresi seçin.",
+  "Recipient's address": "Alıcının adresi",
+  "Advanced settings": "Gelişmiş ayarlar",
+  "Gas amount": "Gaz miktarı",
+  "Gas price": "Gaz fiyatı",
+  "Check": "Kontrol et",
+  "Send": "Gönder",
+  "Password Check": "Şifre kontrolü",
+  "Error while building the transaction": "İşlem oluşturulurken hata oluştu",
+  "Transactions sent!": "İşlemler gönderildi!",
+  "Transaction sent!": "İşlem gönderildi!",
+  "Error while sending the transaction": "İşlem gönderilirken hata oluştu",
+  "Enter your password to send the transaction.": "İşlemi göndermek için şifrenizi girin.",
+  "Lock time": "Kilit süresi",
+  "Duration in minutes after which an idle wallet will lock automatically.": "Boşta beklenirken kaç dakika içinde adresin otomatik olarak kilitleneceği süreç.",
+  "Minutes": "Dakika",
+  "Off": "Kapalı",
+  "Theme": "Tema",
+  "Select the theme and please your eyes.": "Göz zevkinize uygun olan temayı seçin.",
+  "Toggle discreet mode (hide amounts).": "Gizli moda geç (miktarları gizle).",
+  "Password requirement": "Şifre gerekli",
+  "Require password confirmation before sending each transaction.": "Her işlemin gönderilmeden önce şifre doğrulaması gereklidir.",
+  "Password": "Şifre",
+  "Type your password to change this setting.": "Bu ayarı değiştirmek için şifrenizi yazın.",
+  "Enter": "Giriş",
+  "Custom network settings saved.": "Uyarlanmış ağ ayarları kaydedildi.",
+  "Make sure to always check what is the selected network before sending transactions.": "İşlemleri göndermeden önce hangi ağda olduğunuzdan emin olmak için her zaman kontrol edin.",
+  "Network": "Ağ",
+  "Node host": "Düğüm noktası barındırma hizmeti",
+  "Explorer API host": "Tarayıcı API barındırma",
+  "Explorer URL": "URL adresi",
+  "Wallet list": "Cüzdan listesi",
+  "Current wallet": "Şu anki cüzdan",
+  "Wallet name": "Cüzdan ismi",
+  "Lock current wallet": "Şu anki cüzdanı kilitle",
+  "Show your secret recovery phrase": "Gizli kurtarma sözcüklerini göster",
+  "Remove current wallet": "Şu anki cüzdanı kaldır",
+  "Version": "Sürüm",
+  "General": "Genel",
+  "Networks": "Ağlar",
+  "Wallets": "Cüzdanlar",
+  "from": "kimden",
+  "to": "kime",
+  "in": "içinde",
+  "Show in explorer": "Tarayıcıda göster",
+  "From": "Kimden",
+  "Status": "Durum",
+  "Confirmed": "Doğrulandı",
+  "Total value": "Toplam değer",
+  "Click to see more": "Daha fazla görmek için tıklayın",
+  "Click to see less": "Daha az görmek için tıklayın",
+  "Inputs": "Girdiler",
+  "Outputs": "Çıktılar",
+  "New version": "Yeni sürüm",
+  "Version {{ newVersion }} is available. Click \"Update\" to avoid any issues with wallet.": "{{ newVersion }} sürümü mevcut. Cüzdan ile ilgili sorun yaşamamak için \"Update\" (güncelle)'e  tıklayın.",
```

Comment:
I think here it should say 

```
Cüzdan ile ilgili sorun yaşamamak için \"Güncelle\" (güncelle)'e tıklayın.
```

since that's how "Update" translates to based on line 270.

Reply:
I didn' t understand how section work first, now after you showed working position i see that "update" go in base auto translate to "güncelle". So there is no need in extra in here fixed in line string 270